webextension/css/context.css

```diff
@@ -1,19 +1,52 @@
-
-body {
+body, h4 {
     max-height: 100%;
+    font-family: 'Abhaya Libre', serif;
 }
 
-p {
-    font-family:Gill Sans Light;
-    font-size: 20px;
+h1 {
+    font-size: 55px;
+    font-weight: 525;
+    margin: 5vh auto;
 }
 
-.color_code {
-    color:#22c399;
+h4 {
+    font-size: 23px;
 }
 
-.row {
-    margin: auto;
+.color_code, .glyphicon-globe {
+    color: #22c399;
+}
+
+#side-nav-bar {
+    display: none;
+    position: fixed;
+    height: 100vh;
+    width: 23vw;
+    border-right: 3px solid #9A3B38;
+}
+
+.main-content {
+    margin-left: 26vw;
+
+}
+
+.logo, .sidebar-tabs {
+    margin: 3vh 2vw;
+    text-align: center;
+}
+
+.sidebar-tabs {
+    border: 2px solid #9A3B38;
+    background-color: #9A3B38;
+    color: #fff;
+}
+
+.ia-logo {
+    height: 64px;
+}
+
+.wb-logo {
+    height: 100px;
 }
```

Comment:
I'd change this to:
`.wb-logo { width: 100%; padding-top: 1em; }`

If you want me to, I can modify the logo image to make the white area transparent so it looks good in Dark Mode. (or create a new image...)

Reply:
Yes, please make the logo image transparent. That would help for the dark-mode.